templates/cdk.master.auth-webhook.py

```diff
@@ -107,7 +109,7 @@ def check_secrets(token_review):
 
     try:
         output = kubectl(
-            'get', 'secrets', '-o', 'json').decode('UTF-8')
+            'get', 'secrets', '-A', '-o', 'json').decode('UTF-8')
```

Comment:
We *could* go straight for `kube-system` secrets here, but with `-A`, this can be used to validate secrets outside of those created by CK.

Reply:
I don't think this is a good idea and we should limit it to `kube-system`.  As is, this would privilege escalation for any user who can make a secret in any namespace.  Also, if a random secret has a `password` field that happened to match but then didn't have any or valid data in the `username` and `group` fields, it could end up crashing the webhook service (should probably also add some `try` / `except`s anyway).

Limiting the namespace means that RBAC policies could be put in place to ensure normal users can't directly add secrets there and prevent them from escalating their privileges.